src/uTensor/core/TensorMap.hpp

```diff
@@ -1,7 +1,8 @@
 #ifndef UTENSOR_TENSOR_MAP
 #define UTENSOR_TENSOR_MAP
 
-#include <algorithm>
+// #include <algorithm>
```

Comment:
Algorithm library here has been deprecated as the enums are sorted. Can you remove it vs commenting it out?